src/tests/NavBar.test.js

```diff
@@ -0,0 +1,30 @@
+import { render, screen } from "@testing-library/react";
+import NavBar from "../components/NavBar";
+import App from '../components/App/App.component';
+import * as React from 'react';
+
+describe('Tests for checking elements in home page', () => {
+
+    test('Check that search input field is rendered', () => {
+        render(<App />);
+
+        expect(screen.getByPlaceholderText('Search')).toBeInTheDocument();
+    });
+
+    /*test('toogle checkbox for switching modes', () => {
```

Comment:
why are those tests commented? is possible to make them work? or are not necessary?